src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/ov_plugin/core_integration.cpp

```diff
@@ -15,17 +17,19 @@ namespace {
 // OV Class Common tests with <pluginName, deviceName params>
 //
 
-INSTANTIATE_TEST_SUITE_P(ov_plugin_mandatory,
+INSTANTIATE_TEST_SUITE_P(ov_plugin,
                          OVClassModelTestP,
-                         ::testing::Values(ov::test::utils::target_device));
+                         ::testing::Values(ov::test::utils::target_device),
```

Comment:
OVClassModelTestP is being removed in https://github.com/openvinotoolkit/openvino/pull/26197